### ISSUE TITLE
Back Button should not validate

### ISSUE THREAD

**Issue by obiwarn** [cristijora/vue-form-wizard#12]
check your sample:

https://jsfiddle.net/CristiJ/bt5dhqtf/286/

you cannot go back on the second screen, since vaildation block this.

How can I prevent this?

**Comment by cristijora** [comment]
Indeed there is no default way to handle this case. 
https://github.com/cristijora/vue-form-wizard/blob/master/src/components/FormWizard.vue#L322

This is the method that handles tab change when going to the previous tab and I would have to trigger `beforeTabChange` based on some prop.
If you have any suggestions on how to disable/enable validations for certain buttons let me know about it.
If you are fine something like this
```
<form-wizard @on-complete="onComplete" 
                      shape="tab"
                      :validateOnPrev="false"
                      color="#9b59b6">

</form-wizard>
```
I can implement it

**Comment by obiwarn** [comment]
That would be great.

Maybe even default it to `false` since most wizards dont validate on back, I think.

**Comment by obiwarn** [comment]
Or rename the prop to `beforeTabNext` and add `beforeTabPrev` ?

**Comment by cristijora** [comment]
I will play around with more options this weekend and come to a solution for sure.

**Comment by cristijora** [comment]
@obiwarn Please check the initial fiddle. Now back button doesn't validate by default

**Comment by dco5** [comment]
This is exactly what I need! Awesome!

**Comment by obiwarn** [comment]
works! thank you!

**Comment by obiwarn** [comment]
Now a new bug arising from this:
1. go to fiddle
2. go to second step
3. press next > errors appear
4. press back > round icon is still red

**Comment by cristijora** [comment]
Thanks for that. Also noticed that you cannot click on a previous tab/circle without validation

**Comment by cristijora** [comment]
I will take care of this in the following days and wait for your input on this also to see that no issues appear when I do the next release.

**Comment by cristijora** [comment]
This is fixed now. Please verify if it's ok

**Comment by hemnys** [comment]
hi, I have the same problem, please tell me, how to upgrade your package for install the new changes, I have installed using npm

**Comment by cristijora** [comment]
I will make a new npm release soon (today or tomorrow) with the fix for this issue and some other issues

**Comment by hemnys** [comment]
oh nice, Thanks!!!

**Comment by cristijora** [comment]
Released on npm under v0.4.2 https://github.com/cristijora/vue-form-wizard/releases

**Comment by hemnys** [comment]
works fine!!! thanks again

**Comment by obiwarn** [comment]
great work!
there is one little bug left:
when you click on a circle of a previous step (instead of the back button) it still tries to validate on the current step.

**Comment by cristijora** [comment]
Good catch. Time to write unit tests 😄

**Comment by cristijora** [comment]
This might take a little bit more time.

**Comment by cristijora** [comment]
@obiwarn The last part has been addressed in 0.5.0

**Comment by davidnoguerol** [comment]
That's great! Although there is still one final bug.
When you click on a circle of a previous step (instead of the back button) and `:validateOnBack="true"`, it doesn't try to validate on the current step.
Should I open a new issue?
